warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/SplitsFileType.java

```diff
@@ -0,0 +1,47 @@
+package datawave.ingest.mapreduce.job;
+
+/**
+ * Splits Files created will have the form:
+ * 
+ * <pre>
+ * tableA split1 [tablet]
+ * tableA split2 [tablet]
+ * ...
+ * tableB split1 [tablet]
+ * tableB split2 [tablet]
+ * ...
+ * </pre>
+ */
+public enum SplitsFileType {
+    // @formatter:off
+    /*
+     * Indicates that the splits are sampled to a constrained interval
+     * 
+     * For example if there are 10 splits: 
+     * tableA 0ab 
+     * tableA 1cd 
+     * tableA 2ef 
+     * tableA 3gh 
+     * tableA 4ij 
+     * tableA 5kl 
+     * tableA 6mn 
+     * tableA 7op 
+     * tableA 8qr
+     * tableA 9st
+     * 
+     * And the number wanted is 4 then the resulting splits would be 
+     * tableA 2ef 
+     * tableA 4ij 
+     * tableA 6mn 
+     * tableA 8qr
+     * 
+     */
+    // @formatter:on
+    TRIMMEDBYNUMBER,
```

Comment:
Suggestion: to conform to Java's conventions for constants, rename to TRIMMED_BY_NUMBER.  Similarly, SPLITS_AND_LOCATIONS.

Reply:
sounds good